charts/ondat/values.yaml

```diff
@@ -0,0 +1,93 @@
+ondat-operator:
```

Comment:
We should probably add all the possible values here. https://github.com/ondat/charts/blob/umbrella_chart/charts/ondat-operator/values.yaml

Reply:
What do you think about having links to the values.yaml for the respective charts in comments in this values.yaml file? That way, it's fairly easy to know where to go for a full reference!

Reply:
I think having a link to the rest of the values would be a good middleground 🙂